schema/tables.sql

```diff
@@ -577,6 +577,7 @@ CREATE TABLE `BLSample` (
   `blottingDrainTime` int(11) unsigned DEFAULT NULL COMMENT 'Time sample left to drain after blotting, Units: sec',
   `support` varchar(50) DEFAULT NULL COMMENT 'Sample support material',
   `subLocation` smallint(5) unsigned DEFAULT NULL COMMENT 'Indicates the sample''s location on a multi-sample pin, where 1 is closest to the pin base',
+  `isotropy` enum('isotropic','anisotropic') DEFAULT NULL,
```

Comment:
Do you have details on what this refers to?

Reply:
Requirements for i22, page 6 in Mail In System Requirements Draft 3. Should this be in another table?